src/services/VotingService.js

```diff
@@ -0,0 +1,69 @@
+import PollBackendDAO, { EVENT_POLL_VOTED } from 'dao/PollBackendDAO'
+import EventEmitter from 'events'
+import VotingManagerDAO, {
+  EVENT_POLL_ACTIVATED, EVENT_POLL_CREATED, EVENT_POLL_ENDED,
+  EVENT_POLL_REMOVED,
+} from '../dao/VotingManagerDAO'
+
+class VotingService extends EventEmitter {
+
+  constructor () {
+    super(...arguments)
+    this._cache = {}
+  }
+
+  getPollBackendDAO (address) {
```

Comment:
all these methods will be assembled to Base class, so rename its to more generic `getDAO`, `createDAO`, `subscribeToDAO`, `unsibcribeFromDAO`. Avoid `poll`, `manager`, etc